Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'turbolinks'
 gem 'arel', github: 'rails/arel', branch: 'master'
 gem 'mail', github: 'mikel/mail'
 
-gem 'sprockets', '~> 3.0.0.rc.1'
+gem 'sprockets', github: 'rails/sprockets', branch: 'master'
```

Comment:
If we need on generate Gemfiles we need here too. The Railties tests use this Gemfile instead of the generated one.